src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -614,6 +642,7 @@ protected DynamoModel(IStartConfiguration config)
             AuthenticationManager = new AuthenticationManager(config.AuthProvider);
 
             UpdateManager = config.UpdateManager ?? new DefaultUpdateManager(null);
+            HostUpdateManager = (IHostUpdateManager) config.UpdateManager ?? new DefaultUpdateManager(null);
```

Comment:
Can you please put some comment here, why we are casting config.UpdateManager to IHostUpdateManager interface?